x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/use_bulk_actions.ts

```diff
@@ -43,10 +44,15 @@ export function useBulkActions({
     updateBulkActionsState({ action: BulkActionsVerbs.rowCountUpdate, rowCount: alerts.length });
   }, [alerts, updateBulkActionsState]);
 
+  const setIsBulkActionsLoading = (isLoading: boolean = true) => {
```

Comment:
`nitpick` : Should be `useCallback`?

Reply:
I think there won't be a performance gain by adding it